README.rst

```diff
@@ -13,7 +13,11 @@ hnn-core
 	:target: https://codecov.io/gh/jonescompneurolab/hnn-core
 	:alt: Test coverage
 
-This is a leaner and cleaner version of the code based off the `HNN repository <https://github.com/jonescompneurolab/hnn>`_. However, a Graphical User Interface is not supported at the moment in this repository.
+.. image:: doc/images/gui-screenshot.png
```

Comment:
did you add this screenshot? Can you remove it from the commit history? I don't think we should store image files in git. It will unnecessarily bloat the repository.

Reply:
If you want to update the image, just make a comment on github somewhere with the image and use the link from there